internal/gatewayapi/helpers.go

```diff
@@ -208,11 +209,17 @@ func ValidateHTTPRouteFilter(filter *v1beta1.HTTPRouteFilter) error {
 		case string(filter.ExtensionRef.Kind) == egv1a1.KindRateLimitFilter:
 			return nil
 		default:
+			for _, gk := range addlGKs {
+				if filter.ExtensionRef.Group == v1beta1.Group(gk.Group) &&
```

Comment:
validation is essentially being skipped, which is not ideal

Reply:
yeah most validation is usually handled either pre provider (validation webhook) or post provider (related to gateway api logic and status), there are some validations like this one in the provider and we should surface these in the status as well

Reply:
nvm, all the current validations in the provider for external filters will be moved into the validation webhook soon